App/PHP_HTML/login sample/signUp.php

```diff
@@ -62,43 +62,40 @@ function CloseCon($db)
           integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 
     <link rel="stylesheet" type="text/css" href="signUp.css">
+    <link rel="stylesheet" type="text/css" href="style.css">
 
     <style type="text/css">

```

Comment:
You should create file with css for this page rather than having css in this file.